schema/training-module.js

```diff
@@ -1,9 +1,42 @@
 const BaseModel = require('./base-model');
+const { uuid } = require('../lib/regex-validation');
 
 class TrainingModule extends BaseModel {
   static get tableName() {
     return 'trainingModules';
   }
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: [],
+      additionalProperties: false,
+      properties: {
+        id: { type: 'string', pattern: uuid.v4 },
+        'migrated_id': { type: ['string', 'null'] },
+        module: { type: 'string' },
+        species: {
+          type: 'array',
+          items: {
+            type: 'string'
+          }
+        },
+        pass_date: { type: 'string', format: 'date-time' },
+        not_applicable: { type: 'boolean' },
+        accrediting_body: { type: 'string' },
+        other_accrediting_body: { type: 'string' },
+        certificate_number: { type: 'string' },
+        exemption: { type: 'boolean' },
+        exemption_description: { type: 'text' },
```

Comment:
Can we use camelCase for these properties?

Reply:
Sorry yeah that's what I meant - we probably need to do a migration. We can ignore it for now and do it as a separate ticket, but for consistency it would be best not to mix and match.